components/shared/GoogleBtn/GoogleBtn.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { GoogleIcon } from 'public/assets/svgs';
+import { signIn } from 'next-auth/react';
+
+const GoogleBtn = () => {
+  const submitHandler = () => {
```

Comment:
this goes to helpers

Reply:
Google button component or submitHandler?

Reply:
submitHandler